File: tmp/Lv1/문자열을 정수로 바꾸기.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

int solution(string s) {
    int answer = 0;
    if(s[0]=='+'){
        int n=0;
        for(int i=s.size()-1; i>=1; i--)
            answer+=(s[i]-'0')*pow(10,n++);
    }
    else if(s[0]=='-'){
        int n=0;
        for(int i=s.size()-1; i>=1; i--)
            answer+=(s[i]-'0')*pow(10,n++);
        answer*=-1;
    }
    else{
        int n=0;
        for(int i=s.size()-1; i>=0; i--)
            answer+=(s[i]-'0')*pow(10,n++);
    }
    return answer;
}

File: Python/Programmers/Lv1/문자열을 정수로 바꾸기.py
#https://school.programmers.co.kr/learn/courses/30/lessons/12925?language=python3

def solution(s):
    if s[0]=='+':
        answer=int(s[1:len(s)])
    elif s[0]=='-':
        answer=-int(s[1:len(s)])
    else:
        answer=int(s)
    return answer